user_guide/intro_log_in_and_edit_profile.rst

```diff
@@ -0,0 +1,554 @@
+  saved the user settings the file will be checked for consistency with your organization’s rules.  Please contact your 
+  OroCRM administrator for details. The image will be displayed along with your name in the selectors and will help 
+  other users to recognize you quickly.
+
+  |
+  
+  |EditAvatar|
+
+  |
+  
+Any custom details permitted for editing will be available in the *"Additional"* section. In the example above, this is 
+the *"Title"* field.
+
+.. note::
+
+    Although the *"Status"* fields is displayed on the Edit user page, you cannot change it, as it is impossible to 
+    deactivate a profile that you are using.
+
+    
+Manage Your Contact Details
+^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
+ 
+You can define and update the contact information in  the *"General"* section. 
+
+|
+
+.. image:: ./img/intro/user_edit_contacts.png 
+
+|
+  
+- **Your email addresses**: You are required to provide a primary email address. This email address must be unique for 
+  each user in the system. You can also add additional email addresses via :guilabel:`Add Another Email` button. 
+  
+  These addresses are used when an email is created within OroCRM: 
+
+ - The primary email address is used as a default value and additional addresses are available in the selector for the 
+   *"from"* field.
+
+   |
+   
+   |EmailFrom|
+
+   |
+
+.. note::
+
+     If synchronization with an external SMTP server has been enabled for an external mailbox (as described in the 
+     :ref:`Synchronize Your Mailbox with OroCRM section <user-guide-intro-log-in-edit-profile-synchronize>` below, 
+     your addressee will always see the address of the synchronized mailbox as the *"from"* field value.
+
+ - You will see the letters sent by other OroCRM users to your primary and additional email addresses in the 
+   "Activities" section on your User page.
+   
+   |
+   
+   |EmailTo|   
+   
+   |
+   
+- **Phone number**: is displayed to other users among your contact details and will be used as a default value to log a 
+  call related to you or call you via integrated phone applications, such as 
+  :ref:`Google Hangouts <user-guide-hangouts>`.
+
+  |
+  
+  |Phone|
+  
+  |
+  
+- **Email signature**: The signature may be added to any email you write in OroCRM. Your organization settings define 
+  whether the signature will be added automatically or manually. 
+
+
+.. _user-guide-getting-started-profile-password:
+
+Change Your Password
+^^^^^^^^^^^^^^^^^^^^
+
+To change your password, go to the *"Password"* section of the Edit page. 
+
+|
+
+.. image:: ./img/intro/user_edit_password.png
+
+|
+
+You have to:
+
+- Type in your current password
+
+- Type in the new password
+
+- Confirm the new password
+
+
+.. note::
+
+    If you are not using your Google account, nor your corporate-wide credentials, it is strongly recommended to change 
+    your password after the first log-in.
```

Comment:
Edit: If you are not using your Google account, nor your corporate-wide credentials, it is strongly recommended that you change your password after your first login.